VMware.CloudFoundation.PowerManagement.psd1

```diff
@@ -52,15 +52,19 @@
     # Modules that must be imported into the global environment prior to importing this module
     RequiredModules   = @(
         @{
-            ModuleName    = 'Posh-SSH';
+            ModuleName    = 'VMware.PowerCLI'
+            ModuleVersion = '12.5.0'
```

Comment:
Should be bump this to 13.2.0 at a minimum?

Reply:
Yes - I've updated it in the upcoming pull request. This one address only formation and some immediate issues.